config/crd/patches/versionlabel_in_pxc.yaml

```
@@ -3,4 +3,6 @@ kind: CustomResourceDefinition
 metadata:
   name: perconaxtradbclusters.pxc.percona.com
   labels:
-    percona.com/version: v1.18.0
+    app.kubernetes.io/version: v1.18.0
+    app.kubernetes.io/name: percona-xtradb-cluster
+    app.kubernetes.io/part-of: percona-xtradb-cluster
```

maybe:
```suggestion
    app.kubernetes.io/part-of: percona-xtradb-cluster-operator
```
@egegunes WDYT?

---

yes, makes sense

---

https://github.com/percona/percona-xtradb-cluster-operator/pull/2068/commits/55193a6174c714200271d9cf5e4d3f5e8613a8b5